bulbs/videos/templates/videos/series-page.html

```diff
@@ -15,26 +15,67 @@
  <p>Watch The Latest Episode</p>
 </section>
 
-<figure class="video-container">
-  <div class="content">
-    <bulbs-video
-      twitter-handle="{{ twitter_handle }}"
-      data-videohub-base="{{ VIDEOHUB_BASE_URL }}"
-    >
-    </bulbs-video>
-  </div>
-</figure>
+<div class="latest-episode-container">
+  <figure class="video-container">
+    <div class="content">
+      <bulbs-video
+        twitter-handle="{{ twitter_handle }}"
+        data-videohub-base="{{ VIDEOHUB_BASE_URL }}"
+      >
+      </bulbs-video>
+    </div>
+  </figure>
 
-<div id="latest-video-title"></div>
+  {% if channel.slug == 'the-av-club' %}
+    <div id="latest-video-title"></div>
+  </div>
+  {% endif %}
+  {% if channel.slug == 'the-onion' %}
+    <div class="latest-episode-data">
+      <div class="series-title"></div>
+      <hr>
+      <div id="latest-video-title"></div>
+      <div id="latest-video-description"></div>
+      <share-tools
+          class="circle-transparent latest-video-share-tools"
+          share-url={{ external_url }}
+          share-track-action="Single Series: Share">
+        <share-via-facebook icon ></share-via-facebook>
+        <share-via-twitter icon twitter-handle={{twitter_handle}}></share-via-twitter>
+        <share-via-email icon message={{share_message}}></share-via-email>
+      <share-tools>
+    </div>
+  </div>
+  <div class="video-series-ad">
+    <div class="ad-alert">Advertisement</div>
+    {% dfp_ad "non-mobile-leaderboard" %}
+    {% dfp_ad "mobile-only-cube" %}
+  </div>
+{% endif %}
 
 <section class="video-grid">
   <header>
     <h2>Episodes</h2>
+    {% if channel.slug == 'the-onion' %}
```

Comment:
Same comment as above, this should either always be there or a piece that's added by the onion template